devolutions-gateway/src/api/mod.rs

```diff
@@ -27,7 +28,8 @@ pub fn make_router<S>(state: crate::DgwState) -> axum::Router<S> {
         .route("/jet/rdp", axum::routing::get(rdp::handler))
         .nest("/jet/fwd", fwd::make_router(state.clone()))
         .nest("/jet/webapp", webapp::make_router(state.clone()))
-        .nest("/jet/net", net::make_router(state.clone()));
+        .nest("/jet/net", net::make_router(state.clone()))
+        .route("/jet/update", axum::routing::get(update::start_update));
```

Comment:
question: I think this should not be `GET`, but `POST` or `PATCH`. `GET` should be used when fetching some data, not when updating it.